Source/cubfan135/dragndrop.js

```diff
@@ -0,0 +1,84 @@
+/* ======================================================= */
+/* ======= Drag and drop Script by nsandor-webDev ======== */
+/* = This file is called after script.js file and jQuery = */
+/* ========= Can be overwritten toscript.js file ========= */
+/* ======= jQuery downloaded from CDNJS cloudflare ======= */
+/* == Recommended to download jQuery and use it locally == */
+/* ======================================================= */
+
+(function($) {
+    // VARIABLES and CSS objects
+    var body = $('body'),
+        dropZone = body.find('#dropzone'), // Full page overlay drop zone element
+        dropInput = body.find('#bgInput'),  // input file element from right part
+        lastTarget = null,
+        cssHide = { // CSS object to hide the drop zone element
+            visibility: 'hidden',
+            opacity: 0
+        },
+        cssShow = { // CSS object to show the drop zone element
+            visibility: 'visible',
+            opacity: 1
+        },
+        titleFrontSize = {      // Front size for H1 title in drop zone element
+            fontSize: '3em'     // transition effect
+        };
```

Comment:
These variables should be removed as they will not be later used, values are specified as local strings.